jpo-geojsonconverter/src/main/java/us/dot/its/jpo/geojsonconverter/GeoJsonConverterProperties.java

```diff
@@ -35,7 +35,7 @@
 import us.dot.its.jpo.ode.util.CommonUtils;
 
 @ConfigurationProperties("geojsonconverter")
-@PropertySource("classpath:application.properties")
+@PropertySource("classpath:application.yaml")
```

Comment:
Don't think the PropertySource annotation is necessary. the application.yaml file is one of the default property sources.